Add Unified Mode deprecations page (#3097)
chef/chef-web-docs: content/deprecations_unified_mode.md

+++
title = "Deprecation: Enabling Unified Mode (CHEF-33)"
draft = false

gh_repo = "chef-web-docs"
robots = "noindex"

aliases = "/deprecations_unified_mode.html"
+++

> Starting in Chef Infra 17.0 resources should be converted to use Unified Mode, which eliminates the split between

Reviewer: ```suggestion
Starting in Chef Infra Client 17.0 resources should be converted to use Unified Mode, which eliminates the split between
```

the compile and converge modes in the specific resource where it is declared.

> Unified Mode has been available since Chef Infra 14.14.14, Chef Infra 15.3.14 and Chef Infra 16.0 and all core chef resources

Reviewer: ```suggestion
Unified Mode has been available starting with Chef Infra Client 14.14.14, Chef Infra Client 15.3.14, and Chef Infra 16.0 and all core chef resources
```

> Unified Mode has been available starting with Chef Infra Client 14.14.14, Chef Infra Client 15.3.14, and Chef Infra 16.0 and all core chef resources

Reviewer: ```suggestion
Unified Mode has been available starting with Chef Infra Client 14.14.14, Chef Infra Client 15.3.14, and Chef Infra Client 16.0.
```

> in Chef Infra 17.0 have Unified Mode enabled.

Reviewer: No one cares if we implemented it internally. Just nuke that

> in Chef Infra 17.0 have Unified Mode enabled.

Reviewer: ```suggestion
```

Unified Mode is a setting which is local to the resource it is declared inside, and it does not affect
calling recipes or resources, and does not affect any sub-resources which are used.

> The default behavior of Chef Infra 17 is that resources do not run in Unified Mode, which is the same behavior as

Reviewer: ```suggestion
The default behavior of Chef Infra Client 17 is that resources do not run in Unified Mode, which is the same behavior as
```
